Swift/Tests/ArrayTest.swift

```diff
@@ -84,5 +112,120 @@ class ArrayTest: CBLTestCase {
         XCTAssert(mArray3.toArray() == (["Scott", "Daniel", "Thomas"]))
     }
     
+    func testAddMethodsIndividually() throws {
+        let data = arrayOfAllTypes()
+        let array = MutableArrayObject()
+        array.addBoolean(data[0] as! Bool)
+        array.addString(data[1] as? String)
+        
+        array.addInt(data[2] as! Int)
+        array.addInt64(Int64(data[2] as! Int))
+        array.addNumber(NSNumber(integerLiteral: (data[2] as! Int)))
+        
+        array.addDouble(data[3] as! Double)
+        array.addFloat(Float(data[3] as! Double))
+        
+        array.addDate(data[4] as? Date)
+        let dict = MutableDictionaryObject(data: data[5] as! [String: String])
+        array.addDictionary(dict)
+        let subarray = MutableArrayObject(data: data[6] as! [String])
+        array.addArray(subarray)
+        array.addBlob(data[7] as? Blob)
+        array.addValue(NSNull())
```

Comment:
Wondering why not adding NSNull() to the arrayOfAllTypes() as well?

Reply:
when i added, felt like not worth adding, parsing and comparing. But seems like, you are right, adding it to the array is better for consistency, else this will stay as separate and may miss in case a new test is added.

Reply:
I think it's better to add it as it should be consistent when comparing between 2 things.